### ISSUE TITLE
Engine will not run for .py files

### ISSUE THREAD

**Issue by LemmaEOF** [ow2-proactive/jsr223-cpython#17]
Currently `.cpy` is the only extension supported by the script engine. This causes issues with text editor syntax highlighters that are hardcoded to file extensions, and is generally a bit frustrating. It'd be great to have `.py` supported as well.

**Comment by fviale** [comment]
@Boundarybreaker 
Indeed, the file extension .cpy is used to avoid conflict with the jython script engine which already uses the .py extension.
As we do not own the jython engine source code, but use it in our software, we had to use a different extension for the jsr223-cpython engine, in order to dynamically determine which engine is used (cpython or jython) based on file extensions.
